CODEOWNERS

```diff
@@ -7,6 +7,7 @@
 
 # build
 /modules/building/	@konflux-ci/build-maintainers
+/modules/building/pages/prefetching-dependencies.adoc  @ben-alkov
```

Comment:
Having only one memeber in codeowners doesn't scale, it will require all the review from you only and block merging. I'd suggest to create a hermeto team and put the whole team here

Reply:
we can reuse hermeto team also in build definitions

Reply:
Can we simply use `build-maintainers` for the time being?

Reply:
If we do a larger group, I'll need to add in '.*rpm.*md' as well.

Q: Who is in `build-maintainers`?

Reply:
> adding hermeto ppl into build maintainers?
> -    you will be notified about many things you may not care about

True.

> adding build-maintainers into /modules/building/pages/prefetching-dependencies.adoc
> it's already there for the parent dir, so it should be inherited

True, didn't notice it in the diff. This is still a draft and I don't think this is a critical change, so maybe we can setup a prefetch subteam in the GH org and then refresh this PR accordingly?

Reply:
> so maybe we can setup a prefetch subteam in the GH org and then refresh this PR accordingly?

Yeah, this was Martin's idea of setting up a net-new Hermeto team for this, and also build-definitions.